.github/workflows/dynamo_bin_diff.yml

```diff
@@ -72,24 +75,27 @@ jobs:
           {
             "projects": ["src", "test"],
             "sdk": {
-              "version": "8.0.100",
+              "version": "10.0.100-preview.7.25380.108",
```

Comment:
for `build_master` we still need to keep it `8.0.100` for now. Once we merge this PR, we will need to update it to `10.0.100-preview.7.25380.108`